completion/colorjamo.cwl

```diff
@@ -0,0 +1,12 @@
+# colorjamo package
+# ponte-vecchio 2023-01-17 for v.0.3
+
+#include: luacolor
+#include: transparent
```

Comment:
The cwl format is not very flexible when it comes to spaces. This, and every other `#include:`, needs to be
```
#include:luacolor
#include:transparent
```

Reply:
Thanks for pointing this out! Used `grep` to look for all instances of `#include: <pkg>` and fixed them in the new commit.